src/components/safe-apps/hooks/useNativeSwapsAppCard.ts

```diff
@@ -0,0 +1,33 @@
+import { AppRoutes } from '@/config/routes'
+import useChainId from '@/hooks/useChainId'
+import useLocalStorage from '@/services/local-storage/useLocalStorage'
+import type { SafeAppData } from '@safe-global/safe-gateway-typescript-sdk'
+import { SafeAppAccessPolicyTypes } from '@safe-global/safe-gateway-typescript-sdk'
+
+const SWAPS_APP_CARD_STORAGE_KEY = 'showSwapsAppCard'
+
+export function useNativeSwapsAppCard() {
+  const chainId = useChainId()
+  const [isVisible = true, setIsVisible] = useLocalStorage<boolean>(SWAPS_APP_CARD_STORAGE_KEY)
+
+  const swapsCard: SafeAppData = {
+    id: 100_000,
+    url: AppRoutes.swap,
+    name: 'Native swaps are here!',
+    description: 'Experience seamless trading with better decoding and security in native swaps.',
+    accessControl: { type: SafeAppAccessPolicyTypes.NoRestrictions },
+    tags: ['DeFi'],
+    features: [],
+    socialProfiles: [],
+    developerWebsite: '',
+    chainIds: [chainId],
```

Comment:
here I have the card shown on whatever chain is selected. Is swaps enabled on all chains?

Reply:
yes, you can use: `  const isSwapFeatureEnabled = useHasFeature(FEATURES.NATIVE_SWAPS)` to determine if swaps are enabled on the current chain.
Also how about we don't go for a custom "isNativeFeature" flag, but instead expose the swap app id as a custom const NATIVE_SWAP_APP_ID = 100_000
and then later you can compare against that id, instead of taht new isNativeFeature flag.